main.go

```diff
@@ -66,7 +66,13 @@ func getWatchNamespace() (string, error) {
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
+
+        // probeAddr is responsible for the health probe bind address, where the health
+        // probe is responsible for determining liveness, readiness, and configuration 
+        // of the operator pods. Note that the port which is being binded must match
+        // the bind port under './config' and './manifests'
```

Comment:
- Mixing spaces (here) with tabs (code), please use them consistently.
- s/and './manifests'/and './manifests'./  ?
-